src/cleed++/Core/BaseElement.hh

```
@@ -0,0 +1,59 @@
+  virtual std::string getElementName() const;
+  virtual std::string getElementSymbol() const;
+
+  /* setters */
+  virtual void setAtomicNumber(std::size_t Z);
+  virtual void setElementName(const std::string &name);
+  virtual void setElementSymbol(const char symbol[]);
+
+private:
+  std::size_t atomicNumber;
+  std::string name;
+  char symbol[2];
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)